### PR TITLE
test/osd-scrub-dump.sh: fix scrub chunk size

### PR DESCRIPTION
**Pull request by ronen-fr**
The test performs shallow scrubs, intentionally using small chunk sizes to allow dump commands time to check specific details. Following commit ffda6411 (PR#44749), shallow scrubs chunks are controlled by a separate configuration parameter. This PR fixes the test to use the correct parameter.

An additional minor change is an adjustment to the test loop sleep time:
See https://gist.github.com/ronen-fr/e1aaf304ffff7a7c71073b09fde2197e for a sample pre-change run presenting
the timing of replica & local reservations vs the test samples. Note the initial delay in the test sampling.

Fixes: https://tracker.ceph.com/issues/58797

### CODE REVIEW

**Comment by ljflores** [comment]
jenkins test api

**Comment by ljflores** [comment]
jenkins test make check

File: qa/standalone/scrub/osd-scrub-dump.sh [hunk 1]
```diff
@@ -138,7 +138,7 @@ function TEST_recover_unexpected() {
 	    break
 	fi
 	total=$(expr $total + $pass)
-	sleep $(expr $SCRUB_SLEEP \* 2)
+	sleep $(expr $SCRUB_SLEEP \* 2 - 2)
```
Comment (athanatos) [inline qa/standalone/scrub/osd-scrub-dump.sh:141]: ?

**Review by athanatos — APPROVED** [review]
LGTM other than the sleep change -- not sure what the intention is there.

File: qa/standalone/scrub/osd-scrub-dump.sh (hunk 1 above)
Comment (ronen-fr) [reply qa/standalone/scrub/osd-scrub-dump.sh:141]: I've accidentally removed the relevant part in the commit message. The sleeps, esp. the first one, were too long
(as evident from the gist linked in the PR message).
I'll restore the explanation.

Comment (ronen-fr) [reply qa/standalone/scrub/osd-scrub-dump.sh:141]: I actually kept the full explanation in the PR description.

Comment (ronen-fr) [reply qa/standalone/scrub/osd-scrub-dump.sh:141]: ... and simplify the code

**Comment by ronen-fr** [comment]
jenkins retest this please

**Comment by ronen-fr** [comment]
jenkins make check

**Comment by ronen-fr** [comment]
jenkins retest this please

**Comment by ronen-fr** [comment]
jenkins test make check

**Pull request merged**